completion/git-machete.completion.zsh

```diff
@@ -159,6 +159,7 @@ __git_machete_cmds=(
   'add:Add a branch to the tree of branch dependencies'
   'advance:Fast-forward the current branch to match one of its downstreams and subsequently slide out this downstream'
   'anno:Manage custom annotations'
+  'clean:Synchronize with the remote repository: checkout open PRs, delete untracked and unmanaged branches'
```

Comment:
I mean... this description is technically true... but why the command is then called `clean` if it primarly syncs with remote repository? I'd rather put it:

```suggestion
  'clean:Delete untracked and unmanaged branches, and optionally check out open GitHub PRs, '
```

Reply:
(pls reflect in `usage()` and other shell completions if needed)

Reply:
@PawelLipski Can you elaborate on `usage()`, i can't find this function

Reply:
Ouch, sorry, I meant `short_docs` in `git_machete/docs.py:4`... but AFAICS it's all okay:

```
"clean": "Delete untracked and unmanaged branches and also check out user's open GitHub PRs",
```